src/lib/rest/httpRequestSend.cpp

```diff
@@ -406,45 +401,50 @@ int httpRequestSendWithCurl
     {
       // "Switching" protocol https -> http is needed in this case, as CB->Rush request will use HTTP
       protocol = "http://";
-      headerRushHttp = "X-relayer-protocol: https";
```

Comment:
This refactors part of the code related with "regular" headers. Have you tested a full regression with `make test` in oder to check that everything keeps ok?

Reply:
Yes, i did it. It seems to be fine.

Reply:
NTC